app/decorators/user_decorator.rb

```diff
@@ -19,7 +19,8 @@ class UserDecorator < ApplicationDecorator
   ].freeze
 
   def cached_followed_tags
-    follows_map = Rails.cache.fetch("user-#{id}-#{last_followed_at&.rfc3339}/followed_tags", expires_in: 20.hours) do
+    follows_map = Rails.cache.fetch("user-#{id}-#{following_tags_count}-#{last_followed_at&.rfc3339}/followed_tags",
```

Comment:
I added following_tags_count in order to update the cache when we unfollow tags, as the code below does.
https://github.com/forem/forem/blob/83d37df4d79700d3daaad14df560f221689c8226/app/models/user.rb#L313

Reply:
Good catch!